src/App.js

```diff
@@ -1,10 +1,25 @@
-import './App.css';
+import { Heading , Box, Text, Input, InputGroup, HStack, InputRightElement, Icon, Flex   } from '@chakra-ui/react'
+import { FaRegGrinStars, FaSearch, FaArrowDown, FaAngleDown, FaPaperclip, FaGripVertical } from 'react-icons/fa';
+import Card from './components/Card';
 
 function App() {
     return (
-        <div className="App">
-            <h1>Test Text</h1>
-        </div>
+        <Box minHeight="100vh" bg="gray.800" px="75px">
+            <Heading color="white" pt="50px" fontSize='52px'><Icon as={FaRegGrinStars}  w={10} h={10} color='yellow' /> SCTR Teammates</Heading>
```

Comment:
Please extract this whole part into a separate component, we don't want to write jsx for a page in App 

Also, I see this is implemented according to the screenshot from SCTR notion, instead of the wireframe :) it should be based on the wireframe, so please update :)